### ISSUE TITLE
[Request] [Feature] Be able to control both displays with the same rotary 

### ISSUE THREAD

**Issue by gmarquezalvzrez** [simelation/simhub-plugins#15]
Now you can asign he same rotary to both displays, but only the left one is changing the information.
Could be possible to swith both displays at the same time with the same rotary?

Thanks.

**Comment by lukester1975** [comment]
Yeah would be pretty trivial I guess but probably #3 needs implementing first for it to be generally usable, or at least the editing the order of the display modes. Otherwise you just end up with whatever order I hardcoded them which leads to weird pairs of modes, IYSWIM...

**Comment by gmarquezalvzrez** [comment]
Good, thanks!
